packages/tsconfig.json

```diff
@@ -113,6 +116,9 @@
     {
       "path": "./scripts"
     },
+    {
+      "path": "./scripts/src/generate-readme/test/integration/readme-test-adapter"
```

Comment:
@cds95  Did you add this line? I think this is the source of the issues you are seeing.

Reply:
this seems to have come from the EA template.  Removing this for now.